src/js/apps/patients/schedule/filters_app.js

```diff
@@ -15,8 +23,41 @@ export default App.extend({
     this.showFilters();
   },
   showFilters() {
+    this.AllFiltersButtonView();
     this.showGroupsFilterView();
   },
+  AllFiltersButtonView() {
+    if (this.groups.length < 2) return;
+
+    const ownerView = this.showChildView('allFilters', new AllFiltersButtonView());
+
+    this.listenTo(ownerView, 'click', () => {
```

Comment:
can just be `this.listenTo(ownerView, 'click', this.showGroupSidebar);`

Reply:
Yup, that's cleaner. Updated in last push.